Seminar1/task2.py

```diff
@@ -0,0 +1,32 @@
+def check_prime(a):
+    if a > 2:
+        for i in range(2, int(a // 0.5) + 1):
```

Comment:
Зачем int, если оператор целочисленного деления уже есть
```suggestion
        for i in range(2, a // 0.5 + 1):
```
Однако этот подход понизит итерации недостаточно. вспомните теорию чисел.